packages/blade/src/components/Checkbox/_decisions/decisions.md

```diff
@@ -0,0 +1,224 @@
+# Checkbox Decisions <!-- omit in toc -->
+
+- [Design](#design)
+- [Anatomy](#anatomy)
+- [`Checkbox` API](#checkbox-api)
+  - [Examples:](#examples)
+    - [Basic](#basic)
+    - [Validations](#validations)
+- [`CheckboxGroup` API](#checkboxgroup-api)
+  - [Examples:](#examples-1)
+    - [Basic](#basic-1)
+    - [Controlled, Uncontrolled](#controlled-uncontrolled)
+  - [Note on CheckboxGroup](#note-on-checkboxgroup)
+- [Component Architecture](#component-architecture)
+- [Accessibility](#accessibility)
+- [Open Questions](#open-questions)
+
+## Design
+
+[Figma Link](https://www.figma.com/file/jubmQL9Z8V7881ayUD95ps/Blade---Payment-Light?node-id=13227%3A163026) to all variants of the Checkbox component
+
+## `Checkbox` API
+
+| Prop                | Type       | Required | Default   | Description                                                                                                                                              |
+| ------------------- | ---------- | -------- | --------- | -------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| children            | `string`   | No       | undefined | The text to be rendered as CheckboxLabel                                                                                                                 |
```

Comment:
nit(OCD triggering): can you format all the `undefined` to look same? half of them are code formatted other half are not, follow one pattern please? :D

Reply:
Sure.